static/styles/subscriptions.css

```diff
@@ -269,6 +269,42 @@ h4.stream_setting_subsection_title {
                 position: sticky;
                 top: 0;
                 z-index: 1;
+
+                &.active::after,
+                &[data-sort]:hover::after {
+                    content: " \f0d8";
+                    white-space: pre;
+                    display: inline-block;
+                    position: absolute;
+                    padding-top: 3px;
+                    font: normal normal normal 12px/1 FontAwesome;
+                    font-size: inherit;
+                }
+
+                &.active {
+                    opacity: 1;
+                    transition: opacity 100ms ease-out;
+
+                    &.descend::after {
+                        content: " \f0d7";
+                    }
+                }
+
+                &[data-sort]:hover {
+                    cursor: pointer;
+                    background-color: hsla(0, 0%, 95%);
+                    transition: background-color 100ms ease-in-out;
+
+                    &:not(.active)::after {
+                        opacity: 0.3;
+                    }
+                }
+            }
```

Comment:
It seems likely this CSS is copied from somewhere; can we use shared CSS instead?